lib/phoenix_live_view/static.ex

```diff
@@ -54,12 +54,21 @@ defmodule Phoenix.LiveView.Static do
     end
   end
 
-  # TODO: Warn if opts :session has atom keys
   defp load_session(conn_or_socket_session, opts) do
-    user_session = Keyword.get(opts, :session, %{})
+    user_session = validate_session(Keyword.get(opts, :session, %{}))
```

Comment:
You don't need to return the session in this case, you only need to traverse it, which means you don't need to acll things like `Map.new`. It should save on the memory too. :)

Reply:
Thanks!  I thought about that after I told you this was ready.  I'll make that change

Reply:
Done.